### DISCUSSION TITLE
hold-to-confirm activation time

### DISCUSSION THREAD

**Discussion by Inconsist** [ppy/osu#21603] [Ideas]
In my opinion the "hold-to-confirm activation time" should be decoupled from the time required to quick retry a beatmap. 
Globally I'd like to use a setting of 0ms but I also don't want to accidentally retry a beatmap without some activation time threshold to prevent accidental resets. 
So I suggest to either make it into 2 seperate settings or to leave quick retry as it is in stable.

**Comment by peppy** [comment]
Can you describe which global scenario you want the setting of 0ms for? That might help us better understand your use case.

**Reply by Inconsist** [reply]
From my understanding this also affects the time you need to hold down the button when deleting beatmaps quitting out of the game etc. Those I do not want to wait for since they are all reversible actions.

**Reply by 0x84f** [reply]
For me it's not wanting to hold down the exit key to pause a map
